src/reducers/meeting_options_reducer.js

```diff
@@ -0,0 +1,12 @@
+import { FETCH_MEETING_OPTIONS } from '../actions/types';
+
+const INITIAL_STATE = [];
+export default (state = INITIAL_STATE, action) => {
+    switch(action.type){
+        case FETCH_MEETING_OPTIONS:
+            return action.payload.data;
```

Comment:
I may be off base here. But in general, I believe that reducers should not set state directly. Instead it should use overwrite portions of state. Such as:
```
return {
   ...state,
   ...action.payload.data,
};
```

Reply:
Good catch!